### PR TITLE
build: Added support for Node 16

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-3226

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,43 +3,46 @@ name: node_CI
 on:
   push:
     branches:
-      - master
+    - master
   pull_request:
     branches:
-      - "**"
+    - '**'
```
Comment (aht007) [inline .github/workflows/ci.yml:9]: While we are here we can just skip these redundant steps and just mention `pull_request:` as it would do the same

Comment (aht007) [reply .github/workflows/ci.yml:9]: These aren't blockers and I will approve the PR anyway

**Review by mraarif — APPROVED** [review]

**Comment by sarina** [comment]
@binodpant - I'm not sure why you requested my review. Did you mean someone else?

**Review by waheedahmed — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -36,8 +36,8 @@
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-component-cookie-policy-banner": "2.1.14",
-    "@edx/frontend-platform": "1.14.9",
-    "@edx/paragon": "16.14.1",
+    "@edx/frontend-platform": "^1.15.2",
```
Comment (waheedahmed) [inline package.json:39]: nit: remove caret sign from all these requirements.

**Pull request merged**